DTS-STN/DECD-Design-System
# Ds 182 header html css

# DS-182-Curam Header

Main Reviewer: @AnthonyA44 

#### Description & Background Context:

Basic HTML & CSS for the Header component based on the following design in Figma: https://www.figma.com/file/dFM3ltQahFIfvzHB7ftHvl/Sandbox---Design-System?node-id=727%3A3658 

#### Where should the reviewer start?

File name: header.html, header.css

#### How should this be tested?

The reviewer will need to pull the branch onto their local machine and open up the header.html file in their chosen browser.

To test the hiding/showing of menus/change of button colors, change the aria-expanded from "true" to "false" on the following lines of code: 

line 76 (test mobile and desktop)
line 57 (test mobile, this button should not show up on desktop)

#### Are there any breaking changes?
No
